src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -649,22 +640,6 @@ private static ReflectionXmlSerializationReaderHelper.SetMemberValueDelegate Get
             return result;
         }
 
-        private static void SetMemberValue(object o, object value, MemberInfo memberInfo)
```

Comment:
@zhenlan I do no know much serialization code, I assume this is not used by reflection as well?

Reply:
I suspect this is actually needed for reflection based serialization.

@huanwu, can you please review this?

Reply:
I didn't find any place that use this method. It should be fine if all the tests can pass without this method.